torch/_dynamo/utils.py

```diff
@@ -2356,6 +2356,7 @@ def check_numpy_ndarray_args(args, kwargs):
 
 tuple_new = tuple.__new__
 tuple_methods = {method for method in tuple.__dict__.values() if callable(method)}
+list_methods = {method for method in list.__dict__.values() if callable(method)}
```

Comment:
Nit, probably better for another PR, but these global sets are good candidates to become frozen sets